### ISSUE TITLE
ESLint Output not Showing Errors

### ISSUE THREAD

**Issue by al3x-huang** [microsoft/vscode-eslint#1222]
I used to be able to monitor the output tab for ESLint and see any configurations errors but now the output only prints this:
![Screenshot from 2021-03-26 15-28-39](https://user-images.githubusercontent.com/20210402/112700300-14f19480-8e4b-11eb-804c-51992b83c0fd.png)
And that's it. If I were to, for example, mess up a path in `extends` in `.eslintrc.json`, I used to get an error message like...
```
Error: Cannot read config file: /some_path...
Error: ENOENT: no such file or directory, open '/some_path...'
```
But now I can only see these messages if I run ESLint in a terminal via `./node_modules/.bin/eslint`

I used to see these errors in the Output window. How do I get this feature back or is it just discontinued?

I've tried `"eslint.debug": true` in `settings.json` and while it does print more stuff it still doesn't give the error message.

**Comment by dbaeumer** [comment]
The error should still show up when a config file can't be found. Do you have steps on how to reproduce this ?

When you execute eslint in the terminal are you using the workspace folder as the cwd?

**Comment by al3x-huang** [comment]
Reproduction info:
OS: Ubuntu 20.04.01 LTS
VS Code version: 1.54.3 commit 2b9aebd
ESLint extension version: 2.1.19
ESLint version: 7.23.0

1. Launch VS Code
2. Click on `OUTPUT` tab on the bottom bar, `ESLint` on the scrolldown
3. ESLint gives info like
    ```
    [Info  - 10:42:26 AM] ESLint server is starting
    [Info  - 10:42:27 AM] ESLint server running in node v12.18.3
    [Info  - 10:42:27 AM] ESLint server is running.
    [Info  - 10:42:27 AM] ESLint library loaded from: /home/alex/forefront/node_modules/eslint/lib/api.js
    ```
4. However, the output does not give configuration errors.

When run from the root directory of the workspace: `./node_modules/.bin/eslint src/client/App.tsx` the expected error output shows up 
```
Oops! Something went wrong! :(

ESLint: 7.23.0

ESLint couldn't find the config "..." to extend from. Please check that the name of the config is correct.

The config "..." was referenced from the config file in "{src_directory}/.eslintrc.json".

```

**Comment by al3x-huang** [comment]
Oddly enough, when I hover over the "ESLint library loaded from" path and click the `Follow Link (ctrl+click)` option on `/home/alex/forefront/node_modules/eslint/lib/api.js` the error message I expected shows up:
```
Info  - 11:35:09 AM] ESLint server is starting
[Info  - 11:35:09 AM] ESLint server running in node v12.18.3
[Info  - 11:35:09 AM] ESLint server is running.
[Info  - 11:35:09 AM] ESLint library loaded from: /home/alex/forefront/node_modules/eslint/lib/api.js
[Info  - 11:37:39 AM] Failed to load config "..." to extend from. Referenced from: /home/alex/forefront/.eslintrc.json
```

**Comment by dbaeumer** [comment]
Very hard to tell what is going on without being able to reproduce this in our setup. Could you craft a repository I can clone that demos this?

**Comment by prcdpr** [comment]
I can confirm this problem, exactly as described by @al3x-huang

**Comment by dbaeumer** [comment]
Without steps how to reproduce it is vary hard for me to dig into this.

**Comment by al3x-huang** [comment]
This appears to have been fixed in VSCode 1.55.1 commit 08a217c.
Plugins, OS, node packages, etc... versions have not been changed.
